### ISSUE TITLE
Error when uploading Asset Icon from Rotki Windows App into Docker

### ISSUE THREAD

**Issue by Cebrain** [rotki/rotki#3667]
## Problem Definition
I use rotki with docker on latest version and i connect to this instance with my windows installation.

When i try to upload an icon to a self created asset i get an error on my windows maschine.
When i upload it directly in the docker instance (over web browser) it works fine.

Getting error status code 400

### System Description

Operating system: Windows 10
Rotki version: 1.21.3
Docker: 1.21.3

![image](https://user-images.githubusercontent.com/78339531/139933105-a1caae32-0c43-4587-8dc1-7e383d34c4e1.png)

**Comment by yabirgb** [comment]
Hello @Cebrain! Do you mean that you have the rotki backend running in a docker container and then you try to upload the asset from the app client running in a windows machine?

**Comment by Cebrain** [comment]
Hi @yabirgb 
exactly this. I run the docker instance on my server and access the backend most of the time from my windows maschine with Rotki Client.
A Upload with the client is not possible, but if i use the front end from the docker instance the upload works fine.

**Comment by yabirgb** [comment]
Okey thanks for confirming. Tomorrow I'll replicate your setup to reproduce the issue

**Comment by yabirgb** [comment]
The request fails because the app is trying to get the image from a place that is not reacheable 

```
message: "{\"file\": [\"Given path /home/yabirgb/Pictures/ABeMjgBv_400x400.png does not exist\"]}"
```

![image](https://user-images.githubusercontent.com/5068010/140264911-8de884e0-3ae4-410c-a449-da8cc785ba0e.png)

**Comment by kelsos** [comment]
I think I know what the issue is. The way the application works at the moment this is treated wrongly as a packaged app and it just passes the file instead of uploading.

I need to see if there is an easy way to distinguish between app and app with a remote connection.

**Comment by yabirgb** [comment]
> I think I know what the issue is. The way the application works at the moment this is treated wrongly as a packaged app and it just passes the file instead of uploading.
> 
> I need to see if there is an easy way to distinguish between app and app with a remote connection.

Is not possible detecting the backend being use?

**Comment by kelsos** [comment]
I probably need to keep track of when the backend process is started or not.

If I am not mistaken I start and stop it depending on the settings.

**Comment by yabirgb** [comment]
Fixed by #3675
